src/svg/opentable.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+   width="100"
+   height="100"
+   viewBox="0 0 26.458333 26.458334"
+   version="1.1"
+   id="svg391"
+   xmlns="http://www.w3.org/2000/svg">
+  <g id="layer1">
+    <path
+       style="fill:#000000;fill-rule:evenodd;stroke-width:0.479769;fill-opacity:1"
```

Comment:
`fill` attribute shouldn't be defined, or should be `currentColor`. Otherwise, we won't be able to change the color